src/App.vue

```diff
@@ -5,6 +5,11 @@ export default {
   name: 'App',
   components: {
     Connections
+  },
+  computed: {
+    currentOS() {
+      return process.platform
```

Comment:
This isn't likely to change during runtime. It would be much better to use this as a `const` at the top of the script section.

Reply:
Thanks for the feedback :)